contrib/fsl_integration/sct_plugin.py

```diff
@@ -208,52 +290,703 @@ def __init__(self, parent):
+        img2 = overlayORD[1]
+        rawimg = overlayList[img1].dataSource
+        segimg = overlayList[img2].dataSource
+
+        print('Raw Image:', rawimg,)
+        print('Segmentation:', segimg)
+
+        contrast = self.rbox_contrast.GetStringSelection()
+        base_name = os.path.basename(segimg)
+
+        fname, fext = base_name.split(os.extsep, 1)
+
+        out_name = "{}_labeled.{}".format(fname, fext)
+
+        cmd_line = "sct_label_vertebrae -i {} -s {} -c {}".format(rawimg, segimg, contrast)
+        print('Command:', cmd_line)
+        self.call_sct_command(cmd_line)
+
+        outfilename = os.path.join(os.getcwd(), out_name)
+        image = Image(outfilename)
+        overlayList.append(image)
+
+        opts = displayCtx.getOpts(image)
+        opts.cmap = 'subcortical'
+
+#Motion correction for dwi
+class TabPanelMOCO(SCTPanel):
+    DESCRIPTION = """This tool automatically performs
+    motion correction in the diffusion-weighted imaging. For more information, please refer to the
+    article below.<br><br>
+    <b>Usage</b>:<br>
+    To launch the script, the uploaded images into FSLeyes must respect a sequence.
+    In the Overlay list field, the images order is, from the bottom to the top, the diffusion imaging
+    (dwi) and, if you want, a mask imaging (binary mask to limit voxels considered by the registration metric). 
+    It is not necessary uploading the images in such order, with the arrows is possible to sort them.
+    For more information, please refer to the article below.<br><br>
+    <b>Specific citation</b>:<br>
+    Xu et al.
+    <i>Evaluation of slice accelerations using multiband echo planar imaging at 3 T.
+    (2013)</i>. Neuroimage.
+
+    """
+
+    def __init__(self, parent):
+        super(TabPanelMOCO, self).__init__(parent=parent,
+                                            id_=wx.ID_ANY)
+        button_gm = wx.Button(self, id=wx.ID_ANY,
+                              label="dMRI Moco")
+        button_gm.Bind(wx.EVT_BUTTON, self.onButtonMOCO)
+
+        lbl_mask = ['No', 'Yes']
+        self.rbox_mask = wx.RadioBox(self, label='Select Mask:',
+                                         choices=lbl_mask,
+                                         majorDimension=1,
+                                         style=wx.RA_SPECIFY_ROWS)
+
+        sizer = wx.BoxSizer(wx.VERTICAL)
+        sizer.Add(self.rbox_mask, 0, wx.ALL, 5)
+        sizer.Add(button_gm, 0, wx.ALL, 5)
+        self.sizer_h.Add(sizer)
+        self.SetSizerAndFit(self.sizer_h)
+
+    def onButtonMOCO(self, event):
+
+        overlayORD = displayCtx.overlayOrder
+
+        print('Overlay Order:', overlayORD)
+
+        img1 = overlayORD[0]
+        rawimg = overlayList[img1].dataSource
+        print('Raw Image:', rawimg)
+
+        path_name = os.path.dirname(rawimg)
+        base_name = os.path.basename(rawimg)
+
+        fname, fext = base_name.split(os.extsep, 1)
+
+        bvec_name = "{}_bvecs.txt".format(fname)
```

Comment:
i would indicate in the description what is the expected name for bvecs (and ideally provide the possibility to change it)